diff --git a/src/cli/cli.c b/src/cli/cli.c
index 05707c1..2423726 100644
--- a/src/cli/cli.c
+++ b/src/cli/cli.c
@@ -553,7 +553,7 @@ const char *expand_front_variable(TALLOC_CTX *context, const char *string)
 /* Here follows the support for GCC function instrumentation.  Build
  * with CFLAGS='-finstrument-functions -O0 -g' and LDFLAGS='-rdynamic'
  * to enable this mechanism.  */
-
+#if defined(__GLIBC__)
 static int indent_level = 0;
 
 void __cyg_profile_func_enter(void *this_function, void *call_site) DONT_INSTRUMENT;
@@ -582,3 +582,4 @@ void __cyg_profile_func_exit(void *this_function UNUSED, void *call_site UNUSED)
 	if (indent_level > 0)
 		indent_level--;
 }
+#endif
