diff --git a/src/syscall/enter.c b/src/syscall/enter.c
index c133b72..b91c457 100644
--- a/src/syscall/enter.c
+++ b/src/syscall/enter.c
@@ -26,7 +26,7 @@
 #include <linux/net.h>   /* SYS_*, */
 #include <fcntl.h>       /* AT_FDCWD, */
 #include <limits.h>      /* PATH_MAX, */
-
+#include <string.h>      /* strcpy */
 #include "syscall/syscall.h"
 #include "syscall/sysnum.h"
 #include "syscall/socket.h"
